src/Microsoft.Identity.Client/Platforms/net45/PlatformParameters.cs

```diff
@@ -50,7 +50,5 @@ public PlatformParameters(object ownerWindow)
         /// Gets the owner of the browser dialog which pops up for receiving user credentials. It can be null.
         /// </summary>
         public object OwnerWindow { get; private set; }
-
-        internal bool UseHiddenBrowser { get; set; }
```

Comment:
In net45, I create a different webui implementation class in the factory, but in winrt, there is a simple implementation of webui where silent mode is just a flag on WAB